### PR TITLE
Enable BEEFY `report_fork_voting()`

### PR DESCRIPTION
**Pull request by serban300**
Related to https://github.com/paritytech/polkadot-sdk/issues/4523

Follow-up for: https://github.com/paritytech/polkadot-sdk/pull/5188

### CODE REVIEW

**Comment by serban300** [comment]
bot bench substrate-pallet --pallet=pallet_beefy_mmr
bot bench polkadot-pallet --pallet=pallet_beefy_mmr --runtime=rococo
bot bench polkadot-pallet --pallet=pallet_beefy_mmr --runtime=westend

**Comment by serban300** [comment]
bot bench polkadot-pallet --pallet=pallet_beefy_mmr --runtime=rococo

**Comment by serban300** [comment]
bot bench polkadot-pallet --pallet=pallet_beefy_mmr --runtime=westend

**Comment by serban300** [comment]
bot bench substrate-pallet --pallet=pallet_beefy_mmr

File: Cargo.lock [hunk 1]
```diff
@@ -26727,7 +26737,7 @@ dependencies = [
  "array-bytes",
  "log",
  "parity-scale-codec",
- "polkadot-ckb-merkle-mountain-range",
+ "polkadot-ckb-merkle-mountain-range 0.8.1",
```
Comment (acatangiu) [inline Cargo.lock:26740]: why are there two versions of `sp-mmr-primitives` (and two versions of `polkadot-ckb-merkle-mountain-range`) used as deps? can you bump whatever is left behind?

**Review by acatangiu — APPROVED** [review]
nice!

File: Cargo.lock (hunk 1 above)
Comment (serban300) [reply Cargo.lock:26740]: Looks like a "circular" dependency:

```
$ cargo update -p polkadot-ckb-merkle-mountain-range@0.7.0 --precise 0.8.1
    Updating crates.io index
error: failed to select a version for the requirement `polkadot-ckb-merkle-mountain-range = "^0.7.0"`
candidate versions found which didn't match: 0.8.1
location searched: crates.io index
required by package `sp-mmr-primitives v34.1.0`
    ... which satisfies dependency `sp-mmr-primitives = "^34.1.0"` (locked to 34.1.0) of package `polkadot-sdk v0.7.0`
    ... which satisfies dependency `polkadot-sdk = "^0.7"` (locked to 0.7.0) of package `subxt v0.38.0`
    ... which satisfies dependency `subxt = "^0.38"` (locked to 0.38.0) of package `polkadot-zombienet-sdk-tests v0.1.0 (/media/serban/data/workplace/sources/polkadot-sdk/polkadot/zombienet-sdk-tests)`
```

polkadot-zombienet-sdk-tests -> subxt = "^0.38" -> polkadot-sdk = "^0.7" -> sp-mmr-primitives = "^34.1.0" -> polkadot-ckb-merkle-mountain-range = "^0.7.0"

Comment (acatangiu) [reply Cargo.lock:26740]: can we fix the dep to `polkadot-zombienet-sdk-tests`? where are we using it?

Comment (serban300) [reply Cargo.lock:26740]: We are not using it anywhere, but the project is part of polkadot-sdk and it is listed under members in [`Cargo.toml`](https://github.com/paritytech/polkadot-sdk/blob/f4a196ab1473856c9c5992239fcc2f14c2c42914/Cargo.toml#L240) . I don't think we can remove it.

Comment (acatangiu) [reply Cargo.lock:26740]: Ok

**Comment by acatangiu** [comment]
@serban300 why close this?

**Comment by serban300** [comment]
Oops, I didn't want to close this. I deleted my fork of polkadot-sdk because I was getting some warning every day related to it. I will try to reopen it somehow.

**Comment by serban300** [comment]
> Oops, I didn't want to close this. I deleted my fork of `polkadot-sdk` because I was getting some warning every day related to it. I will try to reopen it somehow.

Had to open a new PR: https://github.com/paritytech/polkadot-sdk/pull/6856

**Pull request closed without merging**